src/main/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFs.java

```diff
@@ -123,6 +125,12 @@
     @Argument(fullName=DROP_SOMATIC_FILTERING_ANNOTATIONS_LONG_NAME, doc = "For input somatic GVCFs (i.e. from Mutect2) drop filtering annotations")
     protected boolean dropSomaticFilteringAnnotations = false;
 
+    /**
+     * By default CombineGVCFs reverts all genotypes to no-calls, but calls can be made if specified
+     */
+    @Argument(fullName = CALL_GENOTYPES_LONG_NAME, doc = "Output called genotypes?", optional = true)
+    protected boolean makeGenotypeCalls = false;
```

Comment:
I think that would add too much flexibility.  There are a lot of different GenotypeAssignmentMethods with very subtle differences and I think this could cause more trouble than it's worth.